nav2_dwb_controller/dwb_planner/include/dwb_planner/trajectory_critic.hpp

```diff
@@ -174,11 +174,11 @@ class TrajectoryCritic
 
 protected:
   std::string name_;
-  CostmapROSPtr costmap_ros_;
+  nav2_local_planner_interface::CostmapROSPtr costmap_ros_;
```

Comment:
This type is not specific to the local planner, if we're going to alias it, it needs to be in the costmap_2d package.

Reply:
Replace with ```std::shared_ptr<nav2_costmap_2d::Costmap2DROS>```.